### PR TITLE
Standardize URLs

### LINKED ISSUE #613: Standardize URLs
URL naming scheme needs to be standardized.

### PR DESCRIPTION
**Pull request by jhult**
Resolves #613

### CODE REVIEW

File: src/blocks/components.rs [hunk 1]
```diff
@@ -769,167 +769,6 @@ pub fn BlocksSection() -> impl IntoView {
     }
 }
 
-const ESTIMATED_NON_TABLE_SPACE_IN_SUMMARY: usize = 290;
-
-#[component]
-pub fn SummaryPageBlocksSection() -> impl IntoView {
```
Comment (jhult) [inline src/blocks/components.rs:775]: `SummaryPageBlocksSection` was almost a dupe of `BlocksSection`. I'm not sure which copy was the most up-to-date.

Comment (n1tranquilla) [reply src/blocks/components.rs:775]: The test failures should help you resolve this.

File: src/root.rs [hunk 2]
```diff
@@ -67,24 +66,25 @@ pub fn Root() -> impl IntoView {
...
+                        <Route path="/commands/internal" view=BlockInternalCommandsTab/>
                         <Route path="/analytics" view=BlockAnalyticsTab/>
-                        <Route path="/*any" view=BlockSpotlightTab/>
                     </Route>
+
                     <Route path="/commands" view=CommandsTabbedPage>
-                        <Route path="/" view=UserCommandsPage/>
-                        <Route path="/user-commands" view=UserCommandsPage/>
-                        <Route path="/internal-commands" view=InternalCommandsTab/>
+                        <Route path="*" view=move || view! { <Redirect path="user"/> }/>
```
Comment (n1tranquilla) [inline src/root.rs:85]: Nice, I didn't know there was a redirect.

**Comment by n1tranquilla** [comment]
The build is passing and it looks good to me @jhult . Shall we remove the "draft" status?

**Review by n1tranquilla — APPROVED** [review]

**Pull request merged**